modules/virt-editing-vm-yaml-web.adoc

```diff
@@ -1,14 +1,24 @@
 // Module included in the following assemblies:
 //
 // * virt/virtual_machines/virt-edit-vms.adoc
+//Establishing conditional expressions
+//
+ifeval::["{context}" == "virt-edit-vms"]
```

Comment:
Maybe I am missing something but do you really need conditionals in this module? The content seems to be hard coded for VMs.

Reply:
The {object} conditional expression is used in the virt-editing-vm-yaml-web.adoc module, therefore retaining it. Removing the  unused conditional expressions.

Reply:
ifeval::["{context}" == "virt-edit-vms"]
:object: virtual machine
endif::[]

ifeval::["{context}" == "virt-editing-vm-template"]
:object: virtual machine template
endif::[]

[id="virt-editing-vm-yaml-web_{context}"]